src/cli.js

```diff
@@ -28,9 +30,17 @@ if (program.init) {
   inquirer.prompt()
     .then((answers) => {
       console.log('Saving options into the configuration file');
-      console.log(JSON.stringify(answers, null, 2));
       Object.keys(answers).forEach(k => configStore.set(k, answers[k]));
+      answers.token = 'hidden';
```

Comment:
😱

Reply:
I changed this line so that we are not displaying the user token after it is saved into the config file. we don't want to do this?

Reply:
We do want this. 👍